src/pages/EnablePayments/TermsStep.js

```diff
@@ -112,11 +114,15 @@ class TermsStep extends React.Component {
                             {this.props.translate('common.error.acceptedTerms')}
                         </Text>
                     )}
+                    {!_.isEmpty(errors) && (
+                        <Text style={[styles.formError, styles.mb2]}>
+                            {_.last(_.values(errors))}
+                        </Text>
+                    )}
                     <Button
```

Comment:
Let's replace the Button component with the FormAlertWithSubmitButton. It's more consistent with all other forms and we can get rid of [these lines
](https://github.com/Expensify/App/blob/86bf03a80b5709b702c4e334fc05679c6e05876e/src/pages/EnablePayments/TermsStep.js#L112-L120)

```js
const bool isButtonDisabled = !this.state.hasAcceptedDisclosure || !this.state.hasAcceptedPrivacyPolicyAndWalletAgreement;
...
<FormAlertWithSubmitButton
    buttonText={this.props.translate('termsStep.enablePayments')}
    onSubmit={() => {
        if (!this.state.hasAcceptedDisclosure
            || !this.state.hasAcceptedPrivacyPolicyAndWalletAgreement) {
            this.setState({error: true});
            return;
        }

        this.setState({error: false});
        BankAccounts.acceptWalletTerms({
            hasAcceptedTerms: this.state.hasAcceptedDisclosure
                && this.state.hasAcceptedPrivacyPolicyAndWalletAgreement,
        });
    }}
    isDisabled={isButtonDisabled || this.props.network.isOffline}
     message={this.state.error ? this.props.translate('common.error.acceptedTerms') : !_.isEmpty(errors) ? _.last(_.values(errors)) : ''}
    isAlertVisible={this.state.error || !_.isEmpty(errors)}
/>
```

<img width="945" alt="Screen Shot 2022-08-22 at 1 46 47 PM" src="https://user-images.githubusercontent.com/12268372/185892563-d282caff-20cf-4508-a74f-f18be95f8236.png">